### ISSUE TITLE
Controller support doesn't work.

### ISSUE THREAD

**Issue by Theguysayhi** [tribalfs/GalaxyMaxHzPub#9]
When using a USB or bluetooth controller, inputs are registered on the system ui, but cannot be recognized in most gaming applications.

The problem only goes away when accessibility is turned off in settings, though this turns itself back on again shortly after (which is another issue I'll report).

Phone used is samesung galaxy Fold 3.

**Comment by Theguysayhi** [comment]
![Screenshot_20211206-064616_Gamepad Tester](https://user-images.githubusercontent.com/22719296/144757512-2b8a960c-d65d-4356-ab78-24f3b3f5dbba.jpg)
![Screenshot_20211206-064503_Gamepad Tester](https://user-images.githubusercontent.com/22719296/144757515-152b4dd8-93e0-4fcc-a654-a2051e9aa9ce.jpg)
See that when the app is inactive, inputs are registered along the Axis, but not when the app is active, they become lost and confused.

**Comment by tribalfs** [comment]
- v7.14.16 fixed bug: unable to turn off accessibility when no feature requiring such is enabled.
- controller access is an android issue - fixed in android 12.
